lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb

```diff
@@ -2688,6 +2699,7 @@ def model_add_psz_ac(model,
 
       # create a diffuser and attach the zone/diffuser pair to the air loop
       diffuser = OpenStudio::Model::AirTerminalSingleDuctUncontrolled.new(model, model.alwaysOnDiscreteSchedule)
+      # DEM: might need to change to: diffuser = OpenStudio::Model::AirTerminalSingleDuctConstantVolumeNoReheat.new(model, model.alwaysOnDiscreteSchedule)
```

Comment:
I'm guessing # DEM are initials?  make it a todo instead.  This may confuse the auto documentation on ruby docs.